server/command.go

```diff
@@ -85,11 +85,61 @@ func getCommand() *model.Command {
 		DisplayName:      "Bitbucket",
 		Description:      "Integration with Bitbucket.",
 		AutoComplete:     true,
+		AutocompleteData: getAutocompleteData(),
 		AutoCompleteDesc: "Available commands: connect, disconnect, todo, me, settings, subscribe, unsubscribe, help",
 		AutoCompleteHint: "[command]",
 	}
 }
 
+func getAutocompleteData() *model.AutocompleteData {
+	bitbucket := model.NewAutocompleteData("bitbucket", "[command]", "Available commands: connect, disconnect, todo, me, settings, subscribe, unsubscribe, help")
+
+	connect := model.NewAutocompleteData("connect", "", "Connect your Mattermost account to your bitbucket account")
```

Comment:
```suggestion
	connect := model.NewAutocompleteData("connect", "", "Connect your Mattermost account to your Bitbucket account")
```

Reply:
@sibasankarnayak Did you make these changes?